### PR TITLE
build(endorsement-system-api): Renaming the service and adding docker image

### PR DESCRIPTION
**Pull request by pshomov**
# \<Description\>

Supports island-is/infrastructure#664

## What

Renaming endorsement-system => endorsement-system-api to match a bit better the naming scheme

## Why

Consistency

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,891 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
```
Comment (eirikurn) [inline CHANGELOG.md:4]: Was this accidental?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "island.is",
-  "version": "0.0.1",
+  "version": "0.0.2",
```
Comment (eirikurn) [inline package.json:3]: Was this accidental?

**Review by eirikurn — APPROVED** [review]

**Comment by eirikurn** [comment]
Since we're renaming this for consistency. Should this be `apps/services/endorsements` instead?

The top-level app folders are meant for larger/standalone projects, or when a project has both frontend and backend and we want to group the two. If this is an internal service that's only exposed through other public projects (like graphql API) I think it should be under services.

TBF, this should be documented somewhere but it's probably not ...

**Review by stefanhar — APPROVED** [review]
Renaming LGTM

**Review by stefanhar — CHANGES_REQUESTED** [review]
On closer inspection we are missing changes to paths inside some files, I'll add those

**Comment by pshomov** [comment]
> Since we're renaming this for consistency. Should this be `apps/services/endorsements` instead?
> 
> 
> 
> The top-level app folders are meant for larger/standalone projects, or when a project has both frontend and backend and we want to group the two. If this is an internal service that's only exposed through other public projects (like graphql API) I think it should be under services.
> 
> 
> 
> TBF, this should be documented somewhere but it's probably not ...

What do say @stefanhar - move the service as @eirikurn is suggesting?

**Comment by stefanhar** [comment]
> > Since we're renaming this for consistency. Should this be `apps/services/endorsements` instead?
> > The top-level app folders are meant for larger/standalone projects, or when a project has both frontend and backend and we want to group the two. If this is an internal service that's only exposed through other public projects (like graphql API) I think it should be under services.
> > TBF, this should be documented somewhere but it's probably not ...
> 
> What do say @stefanhar - move the service as @eirikurn is suggesting?

Fine by me

File: package.json (hunk 2 above)
Comment (pshomov) [reply package.json:3]: yes, sorry will clean up. I think the Changelog experiments have committed locally something.

**Review by stefanhar — APPROVED** [review]

**Pull request merged**